source-update/README.md

```diff
@@ -0,0 +1,9 @@
+# EU eHealthNetwork Digitial COVID Certificate RAT list update script
+
+This repository contains scripts that could be used to update value set with common rapid antigen tests (RAT) referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
+
+Release: 1.3.0
```

Comment:
Can't be 1.3.0 - this is already released.
Now that the valuesets have finally got a place of their own to live, no need tokeep the valueset versioning artificially in lock-step with the DCC Schema versioning. In fact, the whole DCC Schema - Valueset combination has been **designed** so that the Valueset versions can change independent of the Schema version, is a core design principle here.